src/SFA.DAS.AssessorService.Data/StandardRepository.cs

```diff
@@ -91,7 +91,10 @@ public async Task<Standard> GetStandardByStandardReferenceAndVersion(string stan
 
         private async Task<Standard> GetStandardsByStandardReferenceAndVersionInternal(string standardReference, string version)
         {
-            var sql = "SELECT * FROM [Standards] WHERE IFateReferenceNumber = @standardReference AND Version = @version";
+            var sql = "SELECT StandardUId, IfateReferenceNumber, LarsCode, Title, Version, Level, Status, TypicalDuration, MaxFunding, " +
```

Comment:
Better to use literal strings for sql - single line prefixed with @ - as it's easier to copy them and run them later